components/match2/wikis/osu/match_group_input_custom.lua

```diff
@@ -0,0 +1,458 @@
+---
+-- @Liquipedia
+-- wiki=osu
+-- page=Module:MatchGroup/Input/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Opponent = require('Module:Opponent')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local TypeUtil = require('Module:TypeUtil')
+local Variables = require('Module:Variables')
+local DateExt = require('Module:Date/Ext')
+local Streams = require('Module:Links/Stream')
+
+local MatchGroupInput = Lua.import('Module:MatchGroup/Input', {requireDevIfEnabled = true})
+
+local NP_STATUSES = {'skip', 'np', 'canceled', 'cancelled'}
+local ALLOWED_VETOES = {'decider', 'pick', 'ban', 'defaultban', 'protect'}
+local ALLOWED_STATUSES = { 'W', 'FF', 'DQ', 'L', 'D' }
+local MAX_NUM_OPPONENTS = 2
+local MAX_NUM_MAPS = 20
+local DEFAULT_BESTOF = 3
+
+local EPOCH_TIME_EXTENDED = '1970-01-01T00:00:00+00:00'
+
+local NOW = os.time(os.date('!*t'))
+local MATCHTIME = nil
```

Comment:
????

Reply:
I mean you mention its always nil above, or I probably take that as **nil** instead of supposingly **0**?